src/modules/sensors/parameters.cpp

```diff
@@ -162,90 +164,20 @@ int initialize_parameter_handles(ParameterHandles &parameter_handles)
 
 	// These are parameters for which QGroundControl always expects to be returned in a list request.
 	// We do a param_find here to force them into the list.
-	(void)param_find("RC_CHAN_CNT");
-	(void)param_find("RC_TH_USER");
+
 	(void)param_find("CAL_ACC0_ID");
 	(void)param_find("CAL_GYRO0_ID");
 	(void)param_find("CAL_MAG0_ID");
-	(void)param_find("CAL_MAG1_ID");
-	(void)param_find("CAL_MAG2_ID");
-	(void)param_find("CAL_MAG3_ID");
 	(void)param_find("CAL_MAG0_ROT");
+	(void)param_find("CAL_MAG1_ID");
 	(void)param_find("CAL_MAG1_ROT");
+	(void)param_find("CAL_MAG2_ID");
 	(void)param_find("CAL_MAG2_ROT");
+	(void)param_find("CAL_MAG3_ID");
 	(void)param_find("CAL_MAG3_ROT");
 	(void)param_find("CAL_MAG_SIDES");
 
-	(void)param_find("CAL_MAG1_XOFF");
-	(void)param_find("CAL_MAG1_XSCALE");
-	(void)param_find("CAL_MAG1_YOFF");
-	(void)param_find("CAL_MAG1_YSCALE");
-	(void)param_find("CAL_MAG1_ZOFF");
-	(void)param_find("CAL_MAG1_ZSCALE");
-
-	(void)param_find("CAL_MAG2_XOFF");
-	(void)param_find("CAL_MAG2_XSCALE");
-	(void)param_find("CAL_MAG2_YOFF");
-	(void)param_find("CAL_MAG2_YSCALE");
-	(void)param_find("CAL_MAG2_ZOFF");
-	(void)param_find("CAL_MAG2_ZSCALE");
-
-	(void)param_find("CAL_MAG3_XOFF");
-	(void)param_find("CAL_MAG3_XSCALE");
-	(void)param_find("CAL_MAG3_YOFF");
-	(void)param_find("CAL_MAG3_YSCALE");
-	(void)param_find("CAL_MAG3_ZOFF");
-	(void)param_find("CAL_MAG3_ZSCALE");
-
-	(void)param_find("CAL_GYRO1_XOFF");
-	(void)param_find("CAL_GYRO1_XSCALE");
-	(void)param_find("CAL_GYRO1_YOFF");
-	(void)param_find("CAL_GYRO1_YSCALE");
-	(void)param_find("CAL_GYRO1_ZOFF");
-	(void)param_find("CAL_GYRO1_ZSCALE");
-
-	(void)param_find("CAL_GYRO2_XOFF");
-	(void)param_find("CAL_GYRO2_XSCALE");
-	(void)param_find("CAL_GYRO2_YOFF");
-	(void)param_find("CAL_GYRO2_YSCALE");
-	(void)param_find("CAL_GYRO2_ZOFF");
-	(void)param_find("CAL_GYRO2_ZSCALE");
-
-	(void)param_find("CAL_ACC1_XOFF");
-	(void)param_find("CAL_ACC1_XSCALE");
-	(void)param_find("CAL_ACC1_YOFF");
-	(void)param_find("CAL_ACC1_YSCALE");
-	(void)param_find("CAL_ACC1_ZOFF");
-	(void)param_find("CAL_ACC1_ZSCALE");
-
-	(void)param_find("CAL_ACC2_XOFF");
-	(void)param_find("CAL_ACC2_XSCALE");
-	(void)param_find("CAL_ACC2_YOFF");
-	(void)param_find("CAL_ACC2_YSCALE");
-	(void)param_find("CAL_ACC2_ZOFF");
-	(void)param_find("CAL_ACC2_ZSCALE");
-
-	(void)param_find("SYS_PARAM_VER");
-	(void)param_find("SYS_AUTOSTART");
-	(void)param_find("SYS_AUTOCONFIG");
-	(void)param_find("SYS_HITL");
-	(void)param_find("PWM_RATE");
-	(void)param_find("PWM_MIN");
-	(void)param_find("PWM_MAX");
-	(void)param_find("PWM_DISARMED");
-	(void)param_find("PWM_AUX_MIN");
-	(void)param_find("PWM_AUX_MAX");
-	(void)param_find("PWM_AUX_DISARMED");
-	(void)param_find("TRIG_MODE");
-	(void)param_find("UAVCAN_ENABLE");
-	(void)param_find("SYS_MC_EST_GROUP");
-
-	// Parameters controlling the on-board sensor thermal calibrator
```

Comment:
They still work for starting temperature calibration without being here. Why is it necessary to "param_find" them?

Reply:
Yes they still work, but QGC will not show the parameters. It will only show params that are accessed (via `param_find`) during bootup. If you remove them here, they're never accessed during normal bootups.

Reply:
I'm still not seeing the problem with SYS_CAL_GYRO, etc. They aren't touched outside of rcS.
https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rcS#L990

The init script runs, does a param compare, they're marked active and then synced to QGC. If you started through another route (custom init script, linux, or qurt) they aren't compared, and then not synced. Am I missing something?

Reply:
You're right about `SYS_CAL_GYRO`, but I'm refering to these:
```
-	(void)param_find("SYS_CAL_TDEL");
-	(void)param_find("SYS_CAL_TMAX");
-	(void)param_find("SYS_CAL_TMIN");
```
They're not accessed in rcS